.changeset/strong-schools-dance.md

```diff
@@ -0,0 +1,5 @@
+---
+"@justeattakeaway/pie-docs": patch
+---
+
+[Changed] - update code pages to point to webc.pie.design and the snacks portal for a number of components
```

Comment:
Not sure if this has been discussed, but should we point to the fozzie storybook for components that exist there too (until we've fully deprecated them).

Shouldn't hold up this PR at all, but just a thought while reviewing.

Reply:
I think probably best we leave fozzie out - think we agreed that in the ticket :)